moviebot/nlu/user_intents_checker.py

```diff
@@ -334,15 +338,27 @@ def check_reject_intent(
             ]
         ):
             dact.intent = UserIntents.REJECT
-            dact.params = [ItemConstraint("reason", Operator.EQ, "dont_like")]
+            preference = convert_choice_to_preference(
+                RecommendationChoices.DONT_LIKE
+            )
+            dact.params = [
+                ItemConstraint("reason", Operator.EQ, "dont_like"),
+                ItemConstraint("preference", Operator.EQ, preference),
+            ]
         elif any(
             [
                 re.search(r"\b{0}\b".format(pattern), utterance)
                 for pattern in PATTERN_WATCHED
             ]
         ):
             dact.intent = UserIntents.REJECT
-            dact.params = [ItemConstraint("reason", Operator.EQ, "watched")]
+            preference = convert_choice_to_preference(
+                RecommendationChoices.WATCHED
+            )
+            dact.params = [
+                ItemConstraint("reason", Operator.EQ, "watched"),
+                ItemConstraint("preference", Operator.EQ, preference),
```

Comment:
Please create an issue for creating enums for the possible contraints that can be expressed ("reason", "perference", etc.)

Reply:
Follow-up issue #225